Code/Editor/IEditorImpl.cpp

```diff
@@ -273,7 +273,7 @@ void CEditorImpl::UnloadPlugins()
 
 void CEditorImpl::LoadPlugins()
 {
-    CryAutoLock<CryMutex> lock(m_pluginMutex);
+    AZStd::lock_guard<AZStd::mutex> lock(m_pluginMutex);
```

Comment:
Recommendation: The lock_guard can be replaced with a scoped lock

```suggestion
    AZStd::scoped_lock lock(m_pluginMutex);
```